portal-impl/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: ${manifest.bundle.name}
 Bundle-SymbolicName: ${manifest.bundle.symbolic.name}
-Bundle-Version: 17.0.1
+Bundle-Version: 18.0.0
```

Comment:
@julschong You did not add/remove any public method in your changes, you should not need a major version bump for your changes. 
I believe you have this because of the changes made by others which already merged into upstream need it. 
If this happens, you do not need to include these changes to your pull, there might be a ci test failed on your pull because of this, but QA will figure out it is not related to you.

Tina.

Reply:
got it, let me run ant all locally again to make sure this is not a mistake by me.